app/src/main/java/com/onboarding/parkingsystemjava/fragment/DateTimePickerDialog.java

```diff
@@ -0,0 +1,57 @@
+package com.onboarding.parkingsystemjava.fragment;
+
+import android.os.Bundle;
+import android.util.Log;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+import androidx.fragment.app.DialogFragment;
+import com.onboarding.parkingsystemjava.databinding.DateTimePickerReservationFragmentBinding;
+import com.onboarding.parkingsystemjava.listener.DateTimeListener;
+import com.onboarding.parkingsystemjava.mvp.contract.DateTimePickerContract;
+import com.onboarding.parkingsystemjava.mvp.presenter.DateTimePickerPresenter;
+import com.onboarding.parkingsystemjava.mvp.view.DateTimePickerView;
+import com.onboarding.parkingsystemjava.utils.ConstantUtils;
+
+public class DateTimePickerDialog extends DialogFragment {
+    private DateTimePickerContract.DateTimePickerPresenter presenter;
+    private DateTimePickerReservationFragmentBinding binding;
+    private DateTimeListener dateTimeListener;
+
+    private static final String BOOL_TAG = "BOOL_TAG";
```

Comment:
this tag should specify whats the meaning of the bundle atribute not only type.
try something like: START_DATE_FLAG or something

Reply:
in the future you could send more booleans and refactor will be needed

Reply:
done